### PR TITLE
Bug 1907660: Render pod list cell height correctly when pod names are long

### PR DESCRIPTION
**Pull request by debsmita1**
**Fixes:**
https://bugzilla.redhat.com/show_bug.cgi?id=1907660

**Solution Description:**
- replaced the `withUserSettingsCompatibility` HOC with the `useUserSettingsCompatibility` hook

**GIF:** 
fix-cell-height

**Browser conformance:**
- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/console-app/src/components/nodes/NodesPage.tsx
Comment (jerolimov) [inline frontend/packages/console-app/src/components/nodes/NodesPage.tsx:257]: Please keep the default as undefined so that this value are not stored in user settings configmap until the user changes the table config.

File: frontend/public/components/namespace.jsx
Comment (jerolimov) [inline frontend/public/components/namespace.jsx:574]: Please keep the default as undefined so that this value are not stored in user settings configmap until the user changes the table config.

File: frontend/public/components/pod.tsx
Comment (jerolimov) [inline frontend/public/components/pod.tsx:318]: Please keep the default as undefined so that this value are not stored in user settings configmap until the user changes the table config.

File: frontend/public/components/namespace.jsx
Comment (jerolimov) [inline frontend/public/components/namespace.jsx:281]: Please keep the default as undefined so that this value are not stored in user settings configmap until the user changes the table config.

**Review by jerolimov — CHANGES_REQUESTED** [review]
WOW. Impressed that this solves this issue, but it works great. Small request to keep the old default value.

**Review by jerolimov — APPROVED** [review]
Works fine. Tested resizing the window, changing the columns and switching between pages. Doesn't see the rendering-height-lag anymore. :+1: 

/lgtm

**Review by rhamilto — COMMENTED** [review]
LGTM, but will wait on Christian for final approval.

**Pull request merged**
